samples/javascript_nodejs/14.nlp-with-dispatch/.env

```diff
@@ -1,2 +1,18 @@
 MicrosoftAppId=
 MicrosoftAppPassword=
+
+LuisHomeAutomationAppId=
```

Comment:
Does the user still need this information if it's already hooked up to the Dispatch model?

Reply:
Sooo...there's a thread in email discussing the inclusion/exclusion of the .json file that's of the parent luis model that dispatches to the children services that it's hooked up to (Kamran and someone else he looped in) 

Issue with it is that to map to the children, it needs to have app ids (which have been checked in mistakenly from C# sample)

But the idea originally that Kamran wanted was to include the .json of the children so that the user can walk through each individual step of creating the services (luis, qna) individually + dispatch for full explanation of the pieces.

But then also include the parent luis model that dispatches to quickly get the bot running...

Anyways! 
TL;DR TBD on what CognitiveModels will actually be included in samples

Reply:
It was decided that the users will simply follow the tutorial. This sample does not have a super fast plug-and-play option, due to the nature of the dispatch jsons needing to map to the children with appIds--which we won't check in

Deleted usused cognitive models & unusued .env variables